users/stewryan/index.md

```diff
@@ -8,26 +8,32 @@ author: Ryan Stewart
 
 For the summer of 2017, I worked in conjunction with Idaho National Lab (INL) and ISU on developing and implementing Functional Expansion Tallies (FETs) in the neutronics code Shift for the Exascale Computing Project. The FETs are meant to create a seamless transition for data from neutronics to computational fluid dynamics codes to allow for fully coupled reactor physics problems to be solved. A short paper on the work can be found [here](./files/Report.pdf).
 
-During my first year at OSU, I explored a small design space in the form of plutonium content for a fixed fissile mass for a prototypic sodium fast reactor. Along with this, I worked/interned at TerraPower, LLC in the field of neutronics and explored how to optimize a plutonium burner given constraints on the plutonium isotopics, disposition rate, and other various reactor physics parameters. I am currently working on novel optimization schemes for sodium fast reactors.
+During my time at OSU, I have examined the material and geometric design space for a small prototypic sodium fast test reactor.
+
+I am currently working on applying multi-objective optimization analysis for determine experimental constraints for a prototypic sodium fast test reactor.
 
 Feel free to email me at stewryan@oregonstate.edu.
 
 ### Research Interests
 1. Fast Reactor Core Analysis
-2. Reactor Physics
-3. Reactor Benchmarking
-
-### Publications 
-1. R. Stewart, and T. Palmer, “Metallic Fuel Design Space for Sodium Fast Reactors,” Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1116-1119.
-2. D. LaBrier, R. Stewart, et al., “Design of an In-Pile Experimental Loop for Fast Test Reactor Conditions,” Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1271-1274.
-3. E. Lum, C. Pope, R. Stewart, et al. "Evaluation of Run 138B at Experimental Breeder Reactor II, A Prototypic Liquid Metal Fast Breeder Reactor," International Handbook of Evaluated Reactor Physics Benchmark Experiments, OECD, 2018.
-4. R. Stewart, C. Pope, and E. Ryan, "Fast Spectrum Reactor Fuel Assembly Sensitivity Analysis," Annals of Nuclear Energy, vol. 110, 2017, pp. 1091-1097.
+2. Multi-Objective Optimization
+3. Applications of Machine Learning to Reactor Physics
+
+### Publications
+1. R. Stewart, E. Lum, and C. Pope, “Design of an experimental breeder reactor run 138B reactor	physics benchmark evaluation management application,” Journal of Nuclear Science and Technology, pp. 1-12, 2019, doi:10.1080/00223131.2019.1680325.
+2. R. Stewart, “FRIDGe: Fast Reactor Input Deck Generator,” Journal of Open Source Software, 4 (40), 1489. https://doi.org/10.21105/joss. 01486.
+3. R. Stewart, and T. Palmer, “Geometric Design Space for Sodium Fast Reactors,”	Transactions of 	the American Nuclear Society, vol. 121, 2019, pp. ***-***.
```

Comment:
White space

Reply:
What white space are you talking about? There doesn't appear to be anything extra between lines 30 and 31. I might be missing something.

Reply:
It may be fine. Lets see